docs/_config.yml

```diff
@@ -43,7 +43,7 @@ plugins:
 #   - vendor/ruby/
 
 # Document versioning
-version: master
+version: v0.12
 display_version_list: true
```

Comment:
so we don't have to mention it like v0.12.0 right?

Reply:
yup

Reply:
thanks @ArangoGutierrez . removing the hold
/hold cancel